tests/runners/browser_test_runner_tests.js

```diff
@@ -417,6 +417,27 @@ describe('browser test runner', function() {
     });
   });
 
+  describe('additionalBrowserSocketEvents', function() {
+    it('binds additional browser socket events', function() {
+      let eventName = 'testEvent';
+      let eventFn = function() {
+        return 'testing additionalBrowserSocketEvents.';
+      }
+      let additionalBrowserSocketEvents = {};
+      additionalBrowserSocketEvents[eventName] = eventFn;
+
+      let socket = new FakeSocket();
+      let reporter = new FakeReporter();
+      let config = new Config('ci', { additionalBrowserSocketEvents, reporter: reporter });
```

Comment:
should this be 
`let config = new Config('ci', { additional_browser_socket_events: additionalBrowserSocketEvents, reporter: reporter });`